projects/pt1/python/torch_mlir_e2e_test/linalg_on_tensors_backends/refbackend.py

```diff
@@ -199,7 +204,9 @@ def compile(self, imported_module: Module):
 
         run_pipeline_with_repro_report(
             imported_module, LOWERING_PIPELINE,
-            "Lowering Linalg-on-Tensors IR to LLVM with RefBackend")
+            "Lowering Linalg-on-Tensors IR to LLVM with RefBackend",
+            enable_ir_printing=False,
```

Comment:
Is this intended?

Reply:
it was (unless you object), since it makes debugging so much quicker (change the False to True ;-)
Otherwise you will have to remember the flag ;-)

Reply:
Okay